src/test/kotlin/com/hibob/academy/employeeFeedback/dao/FeedbackDaoTest.kt

```diff
@@ -0,0 +1,78 @@
+package com.hibob.academy.employeeFeedback.dao
+
+import com.hibob.academy.utils.BobDbTest
+import org.jooq.DSLContext
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import java.sql.Date
+
+@BobDbTest
+class FeedbackDaoTest@Autowired constructor(private val sql: DSLContext){
+    private val feedbackDao = FeedbackDao(sql)
+    private val feedbackTable = FeedbackTable.instance
+    private val companyId = 1
+
+    @Test
+    fun `Submit a new feedback successfully`() {
+        val newFeedback  = FeedbackForSubmission(12, companyId, AnonymityType.IDENTIFIED, "I'm very happy at my workspace!")
+        val feedbackId = feedbackDao.submitFeedback(newFeedback)
+        val expected = listOf(FeedbackInfo(feedbackId, newFeedback.employeeId, newFeedback.companyId, Date.valueOf("2024-09-23"), newFeedback.anonymity, false, newFeedback.feedback))
+        val actual = feedbackDao.viewAllSubmittedFeedback(newFeedback.companyId)
+        assertEquals(expected, actual)
+    }
+
+    @Test
+    fun `View all submitted feedback`(){
+        val newFeedback1  = FeedbackForSubmission(12, companyId, AnonymityType.IDENTIFIED, "I'm very happy at my workspace!")
+        val feedbackId1 = feedbackDao.submitFeedback(newFeedback1)
+        val feedback1 = FeedbackInfo(feedbackId1, newFeedback1.employeeId, newFeedback1.companyId, Date.valueOf("2024-09-23"), newFeedback1.anonymity, false, newFeedback1.feedback)
+
+        val newFeedback2  = FeedbackForSubmission(13, companyId, AnonymityType.IDENTIFIED, "I'm treated very well")
+        val feedbackId2 = feedbackDao.submitFeedback(newFeedback2)
+        val feedback2 = FeedbackInfo(feedbackId2, newFeedback2.employeeId, newFeedback2.companyId, Date.valueOf("2024-09-23"), newFeedback2.anonymity, false, newFeedback2.feedback)
+
+        val expected = listOf(feedback1, feedback2)
+        val actual = feedbackDao.viewAllSubmittedFeedback(companyId)
+        assertEquals(expected, actual)
+    }
+
+    @Test
+    fun `View feedback from a different company than mine`(){
```

Comment:
This test maybe relevant for the service, but not in dao.
In the dao maybe you are testing - view feedbacks of company without feedbacks, something like that.

you are not checking permissiosn here, so i would rename the test